runelite-client/src/main/java/net/runelite/client/plugins/ExternalPluginManager.java

```diff
@@ -465,62 +466,44 @@ else if (!pluginTypes.contains(pluginDescriptor.type()))
 
 		final long start = System.currentTimeMillis();
 
-		// some plugins get stuck on IO, so add some extra threads
-		ExecutorService exec = Executors.newFixedThreadPool(Runtime.getRuntime().availableProcessors() * 2,
-			new ThreadFactoryBuilder()
-				.setNameFormat("external-plugin-manager-%d")
-				.build());
-
-		try
+		List<Plugin> scannedPlugins = new CopyOnWriteArrayList<>();
+		sortedPlugins.forEach(group ->
 		{
-			List<Plugin> scannedPlugins = new CopyOnWriteArrayList<>();
-			sortedPlugins.forEach(group ->
-			{
-				List<Future<?>> curGroup = new ArrayList<>();
-				group.forEach(pluginClazz ->
-					curGroup.add(exec.submit(() ->
-					{
-						Plugin plugininst;
-						try
-						{
-							//noinspection unchecked
-							plugininst = instantiate(scannedPlugins, (Class<Plugin>) pluginClazz, init, initConfig);
-							scannedPlugins.add(plugininst);
-						}
-						catch (PluginInstantiationException e)
-						{
-							log.warn("Error instantiating plugin!", e);
-							return;
-						}
-
-						loaded.getAndIncrement();
-
-						RuneLiteSplashScreen.stage(.67, .75, "Loading external plugins", loaded.get(), scannedPlugins.size());
-					})));
-				curGroup.forEach(future ->
+			List<Future<?>> curGroup = new ArrayList<>();
+			group.forEach(pluginClazz ->
+				curGroup.add(executorService.submit(() ->
 				{
+					Plugin plugininst;
 					try
 					{
-						future.get();
+						//noinspection unchecked
+						plugininst = instantiate(scannedPlugins, (Class<Plugin>) pluginClazz, init, initConfig);
+						scannedPlugins.add(plugininst);
 					}
-					catch (InterruptedException | ExecutionException e)
+					catch (PluginInstantiationException e)
```

Comment:
Looks weird on the diff with whitespaces included, but this catch wasn't actually changed (see below)